### DISCUSSION TITLE
سناریو متفاوت

### DISCUSSION THREAD

**Discussion by r0zbeh** [kianahmadian/python-bale-bot#98] [Q&A] [answered]
سلام و عرض ادب

میشه در مورد این سناریو که ما بیاییم با پایتون یه کد بنویسیم که از طریق اکانت اصلی مون (نه از طریق بات) به سرور بله وصل بشیم و پیام بفرستیم توضیح بدید که آیا این کار شدنیه و اینکه چطور ممکنه ؟
تااون جایی که من میدونم ما باید ابتدا از طریق get ,post متدها به سرور وصل بشیم و cookie رو دریافت کنیم همچنین یک سری اطلاعات و پارامترهای دیگه هم قابل دریافته بعد از طریق اون با سرور اتصال وب سوکت برقرار کنیم من این مراحل رو انجام دادم اتصال به وب سوکتم هم انجام میشه ولی اطلاعاتی که تو سوکت دریافت میشه ناخواناست و اینکه باچه فرمتی اطلاعات باید ارسال بشه؟

**Comment by kian-ahmadian** [comment, marked as answer]
سلام عزیز وقت بخیر
موردی که می‌فرمایید، مربوط به این کتابخانه نیست؛ بنابراین پیشنهاد میکنم برای دریافت پاسخ سوال به پیوی بنده در پیام رسان تلگرام و یا بله پیام خودتون و ارسال کنید

**Reply by r0zbeh** [reply]
سلام ممنون / لطفا ایدی تلگرام رو بفرمایید

**Reply by kian-ahmadian** [reply]
آیدی تلگرام رو میتونین از پروفایل بنده بردارید
